lib/Rx/Observable/TimerObservable.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Rx\Observable;
+
+use Rx\Observable;
+use Rx\ObserverInterface;
+use Rx\SchedulerInterface;
+
+class TimerObservable extends Observable
```

Comment:
Oh, we already have `Interval` with the delay. So we should probably just slim this one down then. :) 

https://github.com/ReactiveX/RxPHP/blob/master/lib/Rx/Observable/IntervalObservable.php